src/pages/ReimbursementAccount/AddressForm.js

```diff
@@ -119,12 +129,13 @@ function AddressForm(props) {
                 errorText={props.errors.city ? props.translate('bankAccount.error.addressCity') : ''}
                 containerStyles={[styles.mt4]}
             />
-            <View style={styles.mt4}>
+
+            <View style={[styles.mt4, styles.mhn5]}>
                 <StatePicker
                     inputID={props.inputKeys.state}
                     shouldSaveDraft={props.shouldSaveDraft}
                     value={props.values.state}
-                    defaultValue={props.defaultValues.state}
+                    defaultValue={props.defaultValues.state || ''}
```

Comment:
Why is this fallback needed? Can we handle this on `StatePicker` instead?

Reply:
I think this won't matter once this PR is merged https://github.com/Expensify/App/pull/22589

Reply:
Can be marked as resolved